packages/vaadin-virtual-list/test/unlimited-size.test.js

```diff
@@ -0,0 +1,171 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, oneEvent } from '@vaadin/testing-helpers';
+import { Virtualizer } from '../src/virtualizer.js';
+
+describe('unlimited size', () => {
+  let virtualizer;
+  let scrollTarget;
+  let elementsContainer;
+
+  beforeEach(() => {
+    scrollTarget = fixtureSync(`
+      <div style="height: 100px;">
+        <div></div>
+      </div>
+    `);
+    const scrollContainer = scrollTarget.firstElementChild;
+    elementsContainer = scrollContainer;
+
+    virtualizer = new Virtualizer({
+      createElements: (count) => Array.from(Array(count)).map(() => document.createElement('div')),
+      updateElement: (el, index) => {
+        el.index = index;
+        el.id = `item-${index}`;
+        el.textContent = el.id;
+      },
+      scrollTarget,
+      scrollContainer
+    });
+
+    virtualizer.size = 1000000;
+  });
+
+  it('should scroll to a large index', () => {
+    const index = Math.min(virtualizer.size / 2);
```

Comment:
`~~` !== `Math.min`

Reply:
True 😅  That was a brainfart. Should've run the tests before pushing. Thx